app/js/views/user-login.jsx

```diff
@@ -51,33 +52,34 @@ export default class UserLogin extends BaseView {
   }
 
   renderBody() {
-    let boxes = (<div hidden></div>);
+    let boxNodes = null;
 
-    if (this.state.boxes.length > 1) {
-      let value = this.state.value !== null ? 0 : this.state.value;
-      const options = this.state.boxes.map((box, index) => {
-        if (box.clientId === this.foxbox.clientId) {
+    if (this.foxbox.boxes.length > 1) {
+      let value = (!this.state.value) ? 0 : this.state.value;
```

Comment:
__note:__ Follow up from here https://github.com/fxbox/app/pull/120/files#r61107814

If we want box list to be dynamically updated, we should keep ```boxes``` in the state, so that when box list changes we change state and then React re-renders view.